odoo/tests/common.py

```diff
@@ -1722,7 +1722,8 @@ def setUpClass(cls):
             cls.addClassCleanup(cls.registry.leave_test_mode)
 
         ICP = cls.env['ir.config_parameter']
-        ICP.set_param('web.base.url', cls.base_url())
+        # URGENT TODO: check if this is correct
```

Comment:
Nope. It just depends on the user's right. There you probably call it with a user not having administration right. Either add the right ot the user on top of the test or do it with another user